packages/block-editor/src/components/colors/use-colors.js

```diff
@@ -151,6 +187,21 @@ export default function __experimentalUseColors(
 		[ setAttributes, colorConfigs.length ]
 	);
 
+	const detectedBackgroundColorRef = useRef();
+	const BackgroundColorDetector = useMemo(
+		() =>
+			withFallbackStyles( ( node ) => {
```

Comment:
Hi @epiqueras,
It seems we are not using the withFallbackStyles functionality. Would not a useEffect call to retrieve the backgroundColor do that same that this useMemo call?

Reply:
We get the node grabbing and updating for free like this.